packages/server/src/adapters/aws-lambda/getPlanner.ts

```diff
@@ -43,10 +44,9 @@ interface Processor<TEvent extends LambdaEvent> {
 function getHeadersAndCookiesFromResponse(response: Response) {
   const headers = Object.fromEntries(response.headers.entries());
 
-  const cookies: string[] = response.headers
-    .getSetCookie()
-    .flatMap((value) => value.split(','))
-    .map((cookie) => cookie.trim());
+  const cookies: string[] = splitSetCookieString(
+    response.headers.getSetCookie(),
+  ).map((cookie) => cookie.trim());
```

Comment:
I don't think the trim is actually necessary my guess is it used to be for space separation with the comma splitting  but leaving it for now just in case someone was relying on it.